bin/aurelia-cli.js

```diff
@@ -2,6 +2,15 @@
 'use strict';
 const resolve = require('../lib/resolve');
 
+const nodeVersion = process.versions.node;
+const [major, minor] = nodeVersion.split('.');
+if (major < 8 || (major === '8' && minor < 9)) {
```

Comment:
var semver = require("semver")

if (semver.lt(process.versions.node, "8.9.0")) {
...

Reply:
Why do by ourselves if semver got us here :)

Reply:
Thx!
Question: this is a bin file, can it access aurelia-cli dependencies? I will try it.

Reply:
Great! It works.